_redirects/boards-mpfs-sev-kit-sev-kit-user-guide.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: forward
+permalink: /redirects/boards-mpfs-sev-kit-es
+target: https://github.com/polarfire-soc/polarfire-soc-documentation/tree/master/boards/mpfs-sev-kit-es/sev-kit-user-guide
```

Comment:
@praveenkumari30718 shouldn't the target link be: https://github.com/polarfire-soc/polarfire-soc-documentation/tree/master/boards/mpfs-sev-kit-es/sev-kit-user-guide/sev-kit-user-guide.md

i.e. add /sev-kit-user-guide.md

Reply:
Why did you move these 2 files out of the _redirects folder? 
demo-guides-mpfs-sev-h264-demo.md
repo-sev-kit-reference-design.md

Can you move them back in to the _redirects folder.